api/comment_threads.rb

```diff
@@ -1,5 +1,7 @@
 
 get "#{APIPREFIX}/threads" do # retrieve threads by course
+  # "sort_key" parameter will change order of threads returned and so may not always return in order
+  # of most comments to least number of comments.
```

Comment:
Some other things we discovered, and I'm not sure the best place to document:
1. Sorting also sorts by pinned threads first (at least in the case of vote sorting).
2. Sorting is handled via Mongo after searching ElasticSearch.  Not sure if this is worth noting anywhere.

Reply:
Barring a better place to put it, I just added it here for now so it's not completely lost.